.github/workflows/CI.yml

```diff
@@ -10,18 +10,11 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
-      - name: Build data-service
-        run: yarn build-data-service
-
-      - name: Build data-loader
-        run: yarn build-data-service
-
       - name: Build frontend
         run: yarn build-frontend
 
       - name: Test frontend
         run: yarn test-frontend
 
-      # This only runs the unit tests - integration tests in CI would be more expensive
-      - name: Test data-service
-        run: yarn test-data-service
+      - name: Test backend
+        run: yarn test-backend
```

Comment:
```suggestion
        run: yarn test-backend --no-default-features
```

Reply:
Not actually sure if this will work, might need to do something special to pass the parameters through yarn?

Reply:
Ok never mind... yarn passes through the parameters just fine, but --no-default-features appears to only work if you define a target package.

i.e. 
- `cargo test --no-default-features --package dgraph` works (resulting in no tests running, all tests in dgraph package are behind the flag) 
- However `cargo test --all --no-default-features` doesn't work, dgraph tests still run 
- Explicitly defining all the packages also doesn't work... `cargo test --no-default-features --package "dgraph service"`

The internet doesn't seem to have an easy fix, except perhaps virtual workspaces... have you used/done this before or shall I just keep it as is?

Reply:
I had a quick look but agree it doesn't seem to work.
I think it's because the dependancies are supposed to specify if they want to enable default features or not. Seems like that's not possible in workspace members.


https://github.com/rust-lang/cargo/pull/11409
This PR fixes it for workspace dependencies but not for members. Let's leave it as it was.